src/CoreWCF.Primitives/src/CoreWCF/Security/SecuritySessionServerSettings.cs

```diff
@@ -0,0 +1,2526 @@
+        //        //{
+        //        //    result = ServerReliableChannelBinder<IReplyChannel>.CreateBinder(this.ChannelBuilder, remoteAddress, sctFilter, sctPriority, faultMode,
+        //        //        this.CloseTimeout, this.SendTimeout);
+        //        //}
+        //   // }
+        //  //  if (result == null)
+        //    //{
+        //    //    throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new NotSupportedException());
+        //    //}
+
+        //   // result.Open(this.OpenTimeout);
+        // //  SessionInitiationMessageHandler handler = new SessionInitiationMessageHandler(result, this, sessionToken);
+        //   // handler.BeginReceive(TimeSpan.MaxValue);
+        //    return result;
+        //}
+
+        void OnTokenIssued(SecurityToken issuedToken, EndpointAddress tokenRequestor)
+        {
+            this.communicationObject.ThrowIfClosed(); //TODO mark open
+            if (!this.acceptNewWork)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new EndpointNotFoundException(SR.Format(SR.SecurityListenerClosing)));
+            }
+            SecurityContextSecurityToken issuedSecurityContextToken = issuedToken as SecurityContextSecurityToken;
+            if (issuedSecurityContextToken == null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new ArgumentException(SR.Format(SR.SessionTokenIsNotSecurityContextToken, issuedToken.GetType(), typeof(SecurityContextSecurityToken))));
+            }
+            // IServerReliableChannelBinder channelBinder = CreateChannelBinder(issuedSecurityContextToken, tokenRequestor ?? EndpointAddress.AnonymousAddress);
+            MessageFilter messageFilter =  CreateSessionMessageServiceDispatcher(issuedSecurityContextToken, tokenRequestor ?? EndpointAddress.AnonymousAddress);
+            bool wasSessionAdded = false;
+            try
+            {
+                this.AddPendingSession(issuedSecurityContextToken.ContextId, issuedSecurityContextToken, messageFilter);
+                wasSessionAdded = true;
+            }
+            finally
+            {
+                //if (!wasSessionAdded)
+                //{
+                //    channelBinder.Abort();
+                //}
+                if (!wasSessionAdded)
+                {
+                    this.ChannelBuilder.RemoveServiceDispatcher<IReplyChannel>(messageFilter);
+                }
+            }
+        }
+       internal SecurityContextSecurityToken GetSecurityContextSecurityToken(UniqueId sessionId)
+        {
+            if (this.pendingSessions1 != null && this.pendingSessions1.ContainsKey(sessionId))
+                return this.pendingSessions1[sessionId];
+            if (this.pendingSessions2 != null && this.pendingSessions2.ContainsKey(sessionId))
+                return this.pendingSessions2[sessionId];
+            return null;
+        }
+
+        void OnTimer(object state)
```

Comment:
Please fix accessor for all methods in this file. VS can do it for you in one go.